### PR TITLE
Switch PowerShell to TLS1.2

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

File: build/Versions.props [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <!-- Toolset -->
     <RoslynToolsRepoToolsetVersion>1.0.0-beta-62614-04</RoslynToolsRepoToolsetVersion>
     <DotNetCliVersion>2.0.2</DotNetCliVersion>
-    <VSWhereVersion>2.1.4</VSWhereVersion>
+    <VSWhereVersion>2.3.2</VSWhereVersion>
```
Comment (jaredpar) [inline build/Versions.props]: Why did this need to change?

File: build/build.ps1 [hunk 2]
```diff
@@ -21,7 +21,8 @@ Param(
 
 set-strictmode -version 2.0
 $ErrorActionPreference = "Stop"
-
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```
Comment (jaredpar) [inline build/build.ps1]: Is there a doc link that explains the implications of this change?

File: build/Versions.props (hunk 1 above)
Comment (tmat) [reply build/Versions.props]: No particular reason, just getting the latest.

File: build/build.ps1 (hunk 2 above)
Comment (tmat) [reply build/build.ps1]: Other than https://msdn.microsoft.com/en-us/library/system.net.servicepointmanager.securityprotocol(v=vs.110).aspx?

**Pull request merged**